src/Bicep.Cli.IntegrationTests/PublishCommandTests.cs

```diff
@@ -499,6 +499,15 @@ private static IEnumerable<object[]> GetValidDataSetsWithPublishSources()
             }
         }
 
+        private static IEnumerable<object[]> OnlyRegistryDataSets()
+        {
+            foreach (var ds in DataSets.AllDataSets.Where(ds => ds.Name == "Registry_LF"))
+            {
+                yield return new object[] { $"{ds.Name}, registry", ds, "", false };
+                break;
+            }
+        }
```

Comment:
Do you know why this change was neccessary?

Reply:
I wanted to test a specific registry, will be removed in the next iteration,